src/CampaignsAdmin/CampaignDetails.tsx

```diff
@@ -44,9 +44,7 @@ export function CampaignDetails() {
    * Retrieves public uploaded image URL from supabase
    */
   const getUploadedImageURL = async (imagePath: string) => {
-    const { uri, apiKey } = settings.supabase;
-    const supabase = createClient(uri, apiKey);
-    const { publicURL, error } = await supabase.storage.from('location-pictures').getPublicUrl(imagePath);
+    const { publicURL, error } = supabaseClientKey.storage.from('location-pictures').getPublicUrl(imagePath);
```

Comment:
+1 for making code easier to read 👌

Reply:
On line 49 you do: 

```ts
if (publicURL) {
      return publicURL;
    }
```

I think you can just return `publicURL` regardless of whether its null or not